components/block/block.module.scss

```diff
@@ -8,4 +8,38 @@
         box-shadow: none;
         outline: 1px solid black;
     }
+
```

Comment:
I switched out the styles to match the way the Make School example was doing by having color-0 be an empty background square. We can switch these out for our other colors. And I removed the Color enum since those props were removed from the Block component